shared/src/packages/users/libs/validation-schemas/common-auth-fields.ts

```diff
@@ -0,0 +1,23 @@
+import joi from 'joi';
+
+import { UserValidationMessage } from '../enums/user-validation-message.enum.js';
+
+const commonUserFields = {
```

Comment:
Name of the variable should be the same as the name of the file

Reply:
Changed the name of the variable.